users/views_api.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import generics, views
+from . import serializers
+from . import permissions
+
+
+class UsersView(generics.CreateAPIView):
+    serializer_class = serializers.UserSerializer
+
+
+class UserDetailView(generics.RetrieveUpdateAPIView):
+    serializer_class = serializers.UserDetailSerializer
+    permission_classes = (permissions.IsOwner,)
```

Comment:
Can you please make a research about when the permission classes is used? You might need a change here.